internal/stringlist/stringlist.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package stringlist provides type StringList that can be used as a type for
+// commandline flags for providing a slice of strings as input.
+package stringlist
+
+import (
+	"fmt"
+)
+
+// StringList is an alias for slice of strings.
+type StringList []string
```

Comment:
Could []string be used directly?

I'm not immediately seeing the additional utility of having a wrapper class here, though could be missing something.

Reply:
The new type implements [the flag.Value interface](https://golang.org/src/flag/flag.go?s=32508:32557#L291).  I think we cannot retain the []string type and provide the functions required by the flag package.  I am open to considering other ways to achieve this, but there does not seem to be any.